crates/figma_import/src/figma_schema.rs

```diff
@@ -14,9 +14,10 @@
 
 use std::collections::HashMap;
 
+use dc_bundle::legacy_definition::element::color::Color;
+use dc_bundle::legacy_definition::element::geometry::{Rectangle, Vector};
```

Comment:
I think we need to clean this up: we have a few places where `toolkit_schema` uses definitions from `figma_schema`, but the two should be unrelated, because we don't want changes to Figma's API to bleed into changes in our format... but similarly, changes to our format shouldn't break Figma deserialization.

So I think we probably want to have two `Rectangle`, `Vector`, and `Color` definitions and some glue to convert from Figma to Toolkit (or dc_bundle now), rather than having shared definitions regardless of who is sharing with whom...

Reply:
Agreed, but I'd rather those things happen in a separate ticket.